app/Http/Controllers/FrontController.php

```diff
@@ -9,4 +9,12 @@ class FrontController extends Controller
     public function home(){
         return view('pages.home');
     }
+
+    public function homePage_with_background_video(){
+        return view('pages.homePage-with-background-video');
+    }
+
+    public function homePage_with_background_parallaxEffect(){
```

Comment:
Rename function name to homePage_with_background_parallax_effect

Reply:
i rename this text